recipes/molecule-signature/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set package = "molecule-signature" %}
+{% set module = "molsig" %}
+{% set version = "2.0.1" %}
+{% set build = 0 %}
+
+# Minimal versions
+{% set python_min = "3.10" %}
+{% set rdkit_min = "2024.09.3" %}
```

Comment:
The Python one has a special meaning to our tools. Any other jinja doesn't help maintainability and ends up obscuring the recipe. Also, it looks like this one is not used.

```suggestion
```